graph/resolver/schema.resolvers.go

```diff
@@ -650,6 +680,32 @@ func (r *queryResolver) Me(ctx context.Context) (*model.User, error) {
 	return authedUser.User, nil
 }
 
+func (r *queryResolver) TwitterUserAutocompletes(ctx context.Context, query string, discussionID string, invitingParticipantID string) ([]*model.TwitterUserInfo, error) {
+	authedUser := auth.GetAuthedUser(ctx)
+	if authedUser == nil {
+		return nil, fmt.Errorf("Need auth")
+	}
+	/* Sanity checks that prevent users from breaking the anonymicity semantics */
+	authedUserParticipants, err := r.DAOManager.GetParticipantsByDiscussionIDUserID(ctx, discussionID, authedUser.UserID)
+	if err != nil {
+		return nil, err
+	}
+	if authedUserParticipants.Anon != nil && authedUserParticipants.Anon.ID == invitingParticipantID {
```

Comment:
The access control is applyed to autocompletes too. If not, a user could see which twitter users have been invited by another participant, which breaks anonymity.